apps/admin-page-e2e/src/integration/pages/discounts.spec.ts

```diff
@@ -89,3 +89,42 @@ describe('discountsPage', () => {
     cy.get('[data-cy=discounts-enable-btn]').should('have.length', mockDiscounts.length);
   });
 });
+
+describe('create discount dialog', () => {
```

Comment:
You're missing a test for failure scenario (creation request fails)
Should display a snackbar, the buttons should remain enabled and the dialog should stay open